MapboxNavigation/NavigationMapView.swift

```diff
@@ -156,6 +153,7 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
      */
     open var tracksUserCourse: Bool = false {
```

Comment:
Rename this property to `showsUserCourse` to avoid confusion with `userTrackingMode`.